slices/bash.yaml

```diff
@@ -0,0 +1,10 @@
+package: bash
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libtinfo6_libs
+    contents:
+      /bin/bash:
+      /bin/sh: { symlink: /bin/bash }
```

Comment:
Hmm.. this is a tricky one. We should have a symlink to `/bin/bash`, but perhaps including this in `contents` might be a bad idea. Because, chisel does not support conflicting paths across slices/packages, yet. See https://github.com/canonical/chisel/issues/50.
So I am worried including `/bin/sh` might raise this issue. For example, if we were to install `dash` and then create a symlink in a same way, we will have conflicts.
So, perhaps, it's better to drop this from the `contents`, at least for now. We could leave a comment/note to remind ourselves that we need to do this manually after installation.

Reply:
Agreed, removed the `/bin/sh` from contents and added a comment.